lib/rpm_package.py

```diff
@@ -140,13 +140,17 @@ def _load(self):
         super(RPM_Package, self)._load()
         try:
             # load distro files
-            files = self.package_data.get('files').get(
-                self.distro.lsb_name).get(self.distro.version)
-
-            self.build_files = files.get('build_files', None)
-            if self.build_files:
-                self.build_files = os.path.join(
-                    self.package_dir, self.build_files)
+            files = self.package_data.get('files', {}).get(
+                self.distro.lsb_name, {}).get(self.distro.version, {})
```

Comment:
It does not make sense to have different cases in different places, centOS should be written with the same case in the entire code to make things simpler. Therefore, it is better if I either use 'centos' in lib/distro_utils.py or use 'centOS' in all yaml files